deepmd/pt/infer/deep_eval.py

```diff
@@ -187,6 +187,10 @@ def get_dim_aparam(self) -> int:
         """Get the number (dimension) of atomic parameters of this DP."""
         return self.dp.model["Default"].get_dim_aparam()
 
+    def has_default_fparam(self) -> bool:
+        """Check if the model has default frame parameters."""
+        return self.dp.model["Default"].has_default_fparam()
```

Comment:
It seems to me this is a breaking change? Do you consider supporting old models?

Reply:
Old models doesn't export this method (`has_default_fparam`).